website/oauth/models/__init__.py

```diff
@@ -489,7 +491,7 @@ class ApiOAuth2Application(StoredObject):
     name = fields.StringField(index=True, required=True, validate=[string_required, MaxLengthValidator(200)])
     description = fields.StringField(required=False, validate=MaxLengthValidator(1000))
 
-    date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow,
+    date_created = fields.DateTimeField(auto_now_add=timezone.now,
```

Comment:
Also, I didn't notice this the first time. [`auto_now_add` should be a bool.](https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.DateField.auto_now_add)